src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -12,9 +12,12 @@ struct cobj_header *hs[MAX_NUM_SPDS + 1];
 /* The booter uses this to keep track of each comp */
 struct comp_cap_info {
 	struct cos_compinfo *compinfo;
+	/* TODO: figure out a better way to do this. Need one ST_user_cap per linked symb. Currently just hard coding 5. */
+	struct usr_inv_cap   ST_user_caps[5];
```

Comment:
1. make this a `#define`d constant.
2. make it larger, this is trivially going to break for any system of any complexity.

I'd recommend `64` to start, and add an error check if you ever need more.